core/controllers/classroom.py

```diff
@@ -33,9 +33,6 @@ class ClassroomPage(base.BaseHandler):
     def get(self, _):
         """Handles GET requests."""
 
-        if not config_domain.CLASSROOM_PAGE_IS_ACCESSIBLE.value:
```

Comment:
Can we server this directly through webpack now? (not sure!)

Reply:
you mean by `app.yaml` and `app_dev.yaml`?

Reply:
I think not, we have a decorator `@acl_decorators.does_classroom_exists` and also use a url fragment 

```python
    get_redirect_route(
        r'/learn/<classroom_url_fragment>', classroom.ClassroomPage),
```